pcsx2/gui/MainFrame.cpp

```diff
@@ -825,9 +851,9 @@ void MainEmuFrame::initializeRecordingMenuItem(MenuIdentifiers menuId, wxString
 	item.Enable(enable);
 }
 
-void MainEmuFrame::enableRecordingMenuItem(MenuIdentifiers menuId, bool enable)
+void MainEmuFrame::enableRecordingMenuItem(int menuId, bool enable)
 {
-	wxMenuItem& item = *m_menuRecording.FindChildItem(menuId);
+	wxMenuItem& item = *m_menuRecording.FindItem(menuId);
```

Comment:
can i get some context on these changes.  From my perspective using the `enum` added some extra type-safety, and `FindChildItem` makes sense as it's specifically for finding menu sub-menu elements, is this to support finding nested menu elements in the future?